src/corehost/common/pal.h

```diff
@@ -54,12 +54,15 @@
 #if defined(_WIN32)
 #define LIB_PREFIX
 #define MAKE_LIBNAME(NAME) (_X(NAME) _X(".dll"))
+#define BASE_HOST_RID _X("win")
```

Comment:
I hear you. In that case, for Windows and OSX, I would rather hard-code the fallback RIDs to be the last highest RIDs known (i.e. win10-* and osx.10.12-*) so that the best attempt is made at the experience.

Linux will still be linux-x64.

What do you think?